webapp/publisher/views.py

```diff
@@ -13,6 +13,27 @@
 from webapp.decorators import login_required
 from webapp.helpers import get_licenses
 
+SUPPORTED_LANGUAGES = {
```

Comment:
it doens't need to be a global variable. You can put it directly in the function for the endpoint

Reply:
you are right